builder/rootfs/etc/confd/templates/check-repos

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+export ETCD=${ETCD:-$HOST:4001}
```

Comment:
When the builder runs this script, the env var `ETCD` ends up set to this literal string: `http://$HOST:$ETCD_PORT`. That's wrong--it should be interpreted. This may also break the `cleanup` script...